src/components/Button/Button.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import { ReactComponent as PlusIcon } from 'assets/plus-icon.svg';
+import './Button.scss';
+
+const Button = ({ type, icon, buttonRef, children, link, onClick, disabled }) => {
+  const rootURL = new RegExp('huoa.org/')
+  if (rootURL.test(link)) {
+    let match = rootURL.exec(link);
+    link = link.slice(match.index + 8)
+  }
```

Comment:
nice first attempt, so this `link = link.slice(match.index + 8)` is probably not robust enough

Reply:
try something with the url - you have access to `pathname` in the dom - react might have a binding for it not sure

Reply:
for ref document.location.pathname gives you the relative path